### PR TITLE
Rewrite StateGroup docs

### PR DESCRIPTION
**Pull request by sebbert**
- Consolidate State and StateGroup docs into a single article (that lives on StateGroup, linked to in State)
- Add a more tutorial-y introduction
- Add example of using the `TransitionState` action
- Add example of the JavaScript APIs of `State` and `StateGroup`.

This PR contains:
- [ ] Changelog N/A
- [X] Documentation
- [ ] Tests N/A

### CODE REVIEW

**Comment by kusma** [comment]
@sebbert: Please explain (in the commit message) what you did apart from rewriting Alone, rewriting anything doesn't sound like an improvement.

**Comment by sebbert** [comment]
@kusma done

**Comment by sebbert** [comment]
aaand done

File: Source/Fuse.Triggers/State.uno [hunk 1]
```diff
@@ -7,17 +7,9 @@ using Fuse.Scripting;
...
-		A `State` consists of a set of @(Animators) inside a `State` object.
-
-		# Example
-		This is a simple example of a state
-
-			<State>
-				<Rotate Degrees="200" Duration="0.4"/>
-				<Move X="10" Duration="0.4"/>
-			</State>
+		See @StateGroup for documentation.
```
Comment (kusma) [inline Source/Fuse.Triggers/State.uno]: "for more details" instead, perhaps? The user is already looking at documentation.

**Pull request merged**